packages/examples/cvat/exchange-oracle/src/core/config.py

```diff
@@ -167,14 +167,24 @@ class CronConfig:
 
 
 class CvatConfig:
+    # TODO: it looks odd to use cvat_ prefix in class attributes inside CvatConfig
     cvat_url = os.environ.get("CVAT_URL", "http://localhost:8080")
     cvat_admin = os.environ.get("CVAT_ADMIN", "admin")
     cvat_admin_pass = os.environ.get("CVAT_ADMIN_PASS", "admin")
     cvat_org_slug = os.environ.get("CVAT_ORG_SLUG", "")
 
     cvat_job_overlap = int(os.environ.get("CVAT_JOB_OVERLAP", 0))
-    cvat_job_segment_size = int(os.environ.get("CVAT_JOB_SEGMENT_SIZE", 150))
+    cvat_task_segment_size = int(os.environ.get("CVAT_TASK_SEGMENT_SIZE", 150))
     cvat_default_image_quality = int(os.environ.get("CVAT_DEFAULT_IMAGE_QUALITY", 70))
+    cvat_max_jobs_per_task = int(os.environ.get("CVAT_MAX_JOBS_PER_TASK", 10 * 1000))
```

Comment:
What is this restriction?

Reply:
https://docs.google.com/presentation/d/17Hppt-uC_zAT_Qyr1Wiw60nd8ZFo3ZQfCqbKaT6vO_k/edit?usp=sharing

Reply:
Ok, makes sense.